src/unittests/line_intersection.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+
+from src.simple_commander.utils.line_intersection import line_intersection
+
+
+class MainControllerTestCase(unittest.TestCase):
+    def setUp(self):
+        self.unit1 = {'width': 2}
+        self.unit2 = {'width': 2}
+
+    def tearDown(self):
+        del self
```

Comment:
I am not sure that it's good idea to delete "self".
Try delete units like: "del self.unit1 del self.unit2"